docs/tap_hold.md

```diff
@@ -126,27 +126,27 @@ bool get_ignore_mod_tap_interrupt(uint16_t keycode, keyrecord_t *record) {
 
 ## Tapping Force Hold
 
-To enable `tapping force hold`, add the following to your `config.h`: 
+To enable `tapping force hold`, add the following to your `config.h`:
 
 ```c
 #define TAPPING_FORCE_HOLD
 ```
 
-When the user holds a key after tap, this repeats the tapped key rather to hold a modifier key.  This allows to use auto repeat for the tapped key.  
+When the user holds a key after tapping it, the tapping function is repeated by default, rather than activating the hold function. This allows to keep the ability to auto-repeat the tapping function of a dual-role key. `TAPPING_FORCE_HOLD` removes that ability to let the user activate the hold function instead, in the case of holding the dual-role key after having tapped it.
```

Comment:
```suggestion
When the user holds a key after tapping it, the tapping function is repeated by default, rather than activating the hold function. This allows keeping the ability to auto-repeat the tapping function of a dual-role key. `TAPPING_FORCE_HOLD` removes that ability to let the user activate the hold function instead, in the case of holding the dual-role key after having tapped it.
```

Slight re-word, as "This allows to keep" reads awkwardly (not that the original was any better :P)